com.unity.shadergraph/Editor/GraphDeltaRegistry/FunctionDefinitions/StandardDefinitions/Artistic/Filter/FadeTransitionNode.cs

```diff
@@ -0,0 +1,30 @@
+using System.Collections.Generic;
+using static UnityEditor.ShaderGraph.Registry.Types.GraphType;
+
+namespace com.unity.shadergraph.defs
+{
+
+    internal class FadeTransitionNode : IStandardNode
+    {
+        public static FunctionDescriptor FunctionDescriptor => new(
+            1,
+            "FadeTransition",
+            "Fade = saturate(FadeValue*(FadeContrast+1)+(NoiseValue-1)*FadeContrast)",
+            new ParameterDescriptor("NoiseValue", TYPE.Float, Usage.In, new float[] { 0.5f }),//ShaderStageCapability.Fragment 
```

Comment:
I think we can get rid of the requirement that this node be Fragment-specific.  There's nothing about what it's doing that would prevent it from running in the vertex shader.